packages/scene-composer/src/store/slices/SceneDocumentSlice.spec.ts

```diff
@@ -32,6 +33,10 @@ jest.mock('../../../src/utils/entityModelUtils/deleteNodeEntity');
 
 jest.mock('../../../src/utils/entityModelUtils/createNodeEntity');
 
+jest.mock('../../../src/utils/entityModelUtils/sceneUtils', () => {
```

Comment:
I know the pattern was already present in this file, but we're trying to move away from `requireActual` in favor of using mocks. Suggest avoiding this pattern in the future.

Reply:
what's the alternative to that if we only want to mock one function but need to use the real implementation for some other functions in the same file?